tests/index.ts

```diff
@@ -116,36 +102,27 @@ export const linterCheckTest = ({
     const driver = setupDriver(dirname, {}, linterName, preCheck);
 
     // Step 3: Asynchronously run each test
```

Comment:
FYI this does not actually run tests asynchronously: it simply tells mocha that there's async logic in the test case and that it needs to `await` the result of each test case.

[This SO answer](https://stackoverflow.com/a/35534550) goes into more detail, but tldr is that mocha is sequential by design (and you can guarantee which test case an exception was thrown inside). Glancing at mocha-parallel-tests it may help, but not sure. We may need to write our own version of mocha-parallel-tests.

Reply:
The comment itself is out of date 🙃 . I'll update. Based on some discussion with Dylan, I don't think it's natively possible/feasible in jest to run each it() in parallel, just use parallelism at the top level (per test file). I think the slowdown isn't too significant, as long as you don't have a large number of tests in one test file, so I think it's fine to keep as is.